nil pointer error when running "src test -m docker" on srclib-python.

testing tree "testdata/case/python-sample-0": Command [src -v do-all -m docker] in python-sample-0 failed: exit status 2.

Output was:
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x4364c0]

goroutine 9 [running]:
runtime.panic(0x842920, 0xb32d33)
    /home/samer/src/go/src/pkg/runtime/panic.c:278 +0xf7 fp=0xc208047df0 sp=0xc208047d58
runtime.panicstring(0xb32d33)
    /home/samer/src/go/src/pkg/runtime/panic.c:559 +0x163 fp=0xc208047e18 sp=0xc208047df0
runtime.sigpanic()
    /home/samer/src/go/src/pkg/runtime/os_linux.c:238 +0x11e fp=0xc208047e30 sp=0xc208047e18
sourcegraph.com/sourcegraph/srclib/scan.func·001(0x0, 0x0)
    /home/samer/code/go/src/sourcegraph.com/sourcegraph/srclib/scan/scan.go:39 +0x220 fp=0xc208047f48 sp=0xc208047e30
code.google.com/p/rog-go/parallel.func·003()
    /home/samer/code/go/src/code.google.com/p/rog-go/parallel/parallel.go:67 +0x71 fp=0xc208047fa8 sp=0xc208047f48
runtime.goexit()
    /home/samer/src/go/src/pkg/runtime/proc.c:1621 fp=0xc208047fb0 sp=0xc208047fa8
created by code.google.com/p/rog-go/parallel.(*Run).Do
    /home/samer/code/go/src/code.google.com/p/rog-go/parallel/parallel.go:70 +0xf2

--- a/scan/scan.go
+++ b/scan/scan.go
@@ -37,7 +37,7 @@ func ScanMulti(scanners []toolchain.Tool, opt Options, treeConfig map[string]int
 			if err != nil {
 				cmd, newErr := scanner.Command()
 				if newErr != nil {
-					return fmt.Errorf("scanner %v, cmd error: %s", newErr)
+					return fmt.Errorf("cmd error: %s", newErr)
 				}
 				return fmt.Errorf("scanner %v: %s", cmd.Args, err)
 			}

